### DISCUSSION TITLE
Week 1 Thinking Cap: Privacy and Misuse of Data

### DISCUSSION THREAD

**Discussion by gu-ana** [albertkun/21S-ASIAAM-191A#43] [Past Thinking Caps]
I have two pretty different things I wanted to point out in this thinking cap. 

Firstly, Facebook had just leaked half a billion of user's data, which if we total up to be a population, would be number 3 in the world after China and India. But this isn't some one off event that happens once in a blue moon. Equifax had leaked 150 million people's personal data and basically got a slap on the wrist. Google basically tracks every thing you do online and there's nothing that you can do (and no, incognito mode doesn't actually make you incognito, every website you go on is still stored by the ISP regardless). The United States is very behind cyber security laws and 50-80 year old politicians sitting in congress aren't doing anything about it. I think it'll have to up to Gen X and after to really start having substantial change for the revamp of cybersecurity laws. There's talk saying that if Facebook had leaked any EU citizen's data in this data breach, they're gonna get their butt handed to them because the EU has a more updated set of online privacy laws that can prosecute Facebook for mishandling user data.

We've all seen the "Drinking a glass of wine a day can prevent ___!" or "Eating dark chocolate every day can prevent cancer!" titles in the news. As much as I want this to be true I know that it most likely is not. Sure, the data can be "statistically significant" but we have to consider the sample size, how the test was conducted, are the results reproducible, etc..  This can be  expanded to not just food but also political studies that write sensationalized titles to garner clicks and generate engagement. I think at the end of the day, what's most important is that the general population remain critical of the information fed to them online. I will say that I fall victim to sensational headlines in the media and lord knows how much more my parents do as they love sending me articles that are exaggerated egregiously.

**Comment by rh5140** [comment]
It's disappointing (though unsurprising) how common data breaches are, and how we can't trust the companies that have our data even if they don't get breached. This is kinda the other way around from your example, [but David Carroll, a US citizen, got Cambridge Analytica's parent company to plead guilty to breaking UK data protection laws](https://www.wired.com/story/one-mans-obsessive-fight-to-reclaim-his-cambridge-analytica-data/).
> "This prosecution, the first against Cambridge Analytica, is a warning that there are consequences for ignoring the law,” the information commissioner, Elizabeth Denham, said in a statement following the hearing. “Wherever you live in the world, if your data is being processed by a UK company, UK data protection laws apply.”

Another thing I want to highlight in that article is that tech giants like Facebook have no issue with giving away massive amounts of data for profit.
> At the time [2014], the social networking giant allowed developers to slurp up data not just from their own users but from their users’ friends, all without their friends’ awareness or explicit consent. Facebook didn't officially end this policy until April 2015, and continued to give some developers access even after that.

I agree that developing critical media literacy is extremely important, especially in this age where articles can be easily shared with a click/tap through social media platforms. Misinformation can spread extremely quickly that way. I've seen people share news on their Instagram story and adding text like "I'm not sure if this is true or not but..." and it's pretty sad to see that they don't take the time to verify the information before posting it, even when they're already unsure of its accuracy.

**Comment by jason-muljadi** [comment]
> But this isn't some one off event that happens once in a blue moon. 

Thank you for pointing this out. In general, I feel that we as a society are more or less used to having invasions of privacy. Somehow, this is still big news even though that data leaks happen on a micro level every single day. I personally feel that I am used to getting my information leaked even if I do keep it relatively private and that I expect that people are able to know who I am and what I do based on a Google search or two. Privacy is no longer a right, but it becomes a dream. 

I feel like I'm in the minority of people who just accept what this society does with data, but it is still very concerning to have these data leaks. Case in point when there was a cyberattack on Accelerion and many students and faculty members of the UC communities have compromised data that has sensitive information such as financial information and home addresses. This is very alarming. What is also alarming is the continuous threat of doxxing online celebrities and streamers. I understand that a person may say and do things that are very problematic; I would condemn their statement. But the ease in which "regular people" are able to disclose personal information in attempt to harm others do not settle well with me at all. 

> The United States is very behind cyber security laws and 50-80 year old politicians sitting in congress aren't doing anything about it. I think it'll have to up to Gen X and after to really start having substantial change for the revamp of cybersecurity laws. 

This brought a chuckle to me. I definitely do feel that we need to have cyber education. It was hilarious but also embarrassing to see how politicians handled big tech companies and an overall lack of knowledge when it comes to the internet. There needs to be a greater internet education for our society, but disseminating that information is difficult.


> We've all seen the "Drinking a glass of wine a day can prevent ___!" or "Eating dark chocolate every day can prevent cancer!" titles in the news. As much as I want this to be true I know that it most likely is not. Sure, the data can be "statistically significant" but we have to consider the sample size, how the test was conducted, are the results reproducible, etc.. This can be expanded to not just food but also political studies that write sensationalized titles to garner clicks and generate engagement. 

I agree with this so much. The purpose of science articles is to provide plausibility for an alternative hypothesis or, if the experiment's trials were subpar, proving that the null hypothesis is plausible. You cannot really _**prove**_ anything in science 100%. The _**laws**_ of science are still just observations that happen to be generally accepted. With so many confounding factors that may not even be considered, it is difficult to provide an absolute in an observation. Looking at the world more probabilistically seems much more rational. But sensation does sell!

> I think at the end of the day, what's most important is that the general population remain critical of the information fed to them online. I will say that I fall victim to sensational headlines in the media and lord knows how much more my parents do as they love sending me articles that are exaggerated egregiously.

Twitter will always be my downfall, so I also concur with this. The WhatsApp articles for the parents also do not help.
